ResourceStatusSystem/Utilities/CSHelpers.py

```diff
@@ -136,19 +131,6 @@ def getNodes():
 
   return S_OK(nodes)
 
-################################################################################
-
-
-def getStorageElements():
```

Comment:
It is not nice to drop functions in patch releases

Reply:
Yes I agree "it's not nice", but I would be surprised if anybody use it. Do you?

Reply:
One possibility would be that I leave these functions, but I mark them as `@deprecated`, I think this would be acceptable

Reply:
We are not using them, maybe no-one, but `deprecated` is certainly better